.github/workflows/_11_coverage.yml

```diff
@@ -17,10 +17,21 @@ jobs:
       credentials:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
+      env:
+        NSC_CACHE_PATH: "/cache"
+      volumes:
+        - /cache:/cache # Where the Cache Volume is mounted.
+      options: --cap-add=SYS_ADMIN # Required to by nscloud-cache-action to call `mount`.
     steps:
       - name: Checkout 🛒
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
 
+      - name: Setup Rust cache 🦀💰
+        if: ${{ github.event_name == 'pull_request' && github.base_ref == 'main' }}
```

Comment:
I think we can remove these `if` statements - as we added these when debugging but probably wasn't impacting anything